test/pointing_party/vote_calculator_test.exs

```diff
@@ -0,0 +1,84 @@
+defmodule PointingParty.VoteCalculatorTest do
+  use ExUnit.Case, async: true
+  use ExUnitProperties
+
+  describe "calculate_votes/1" do
+    setup do
+      points_map = fixed_map(%{points: integer(1..5)})
+
+      metas_map =
+        fixed_map(%{
+          metas: list_of(points_map, length: 1)
+        })
+
+      users = nonempty(map_of(string(:alphanumeric), metas_map))
+      [users: users]
+    end
+
+    property "winning value is a list or a integer", %{users: users} do
+      check all users <- users do
```

Comment:
:100: for renaming the property parameter from `user` to `users. I discourage people from reassigning values to parameters. (Martin Fowler used to have a refactoring just for this, but reassigning to parameters is now a general case of what he calls ["Split Variable."](https://refactoring.com/catalog/splitVariable.html)) It can create some confusion about what exactly a variable refers to. What would you think about this?
```ex
check all user_map <- users do
```